app-frontend/src/app/components/sceneItem/sceneItem.controller.js

```diff
@@ -1,10 +1,16 @@
 export default class SceneItemController {
-    constructor() {
+    constructor($scope) {
         'ngInject';
+        $scope.$watch(
+            () => this.selected({scene: this.scene}),
+            (selected) => {
```

Comment:
Yeah, it's definitely a debatable thing, and I see your point. We should decide on a preferred style for this when we come up with a style guide, and update the .eslintrc accordingly.